src/models/subscriptiondata.cpp

```diff
@@ -147,6 +147,12 @@ bool SubscriptionData::fromJson(const QByteArray& json) {
     return false;
   }
 
+  // Check if the privacy bundle upgrade is available. For now we use the
+  // currency as a proxy to determine if the upgrade is available for a user.
+  const QStringList bundleUpgradeWhiteList = {"USD", "CAD"};
+  m_bundleUpgradeAvailable = bundleUpgradeWhiteList.contains(m_planCurrency) &&
+                             m_type == SubscriptionWeb;
```

Comment:
Can you invert the conditions? First a boolean check and then the list?

Reply:
Sure!